Documentation/compatibility/wpf-accessibility-improvements.MD

```diff
@@ -0,0 +1,101 @@
+## 195: Accessibility improvements in WPF
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+__High Contrast improvements__
+
```

Comment:
"improvements" and "fixed" are not fully compatible, and a fix generally doesn't require a compatibility switch. It would be best to rephrase these to focus on the change and the previous behavior. For example, instead of "Focus for expander was not visible. This is now fixed.", substitute the following:
Starting with the .NET Framework 4.7.1, the focus for the expander is visible. In previous versions of the .NET Framework, it was not.
This applies to each of the items that has been fixed.